signer/src/context/messaging.rs

```diff
@@ -22,6 +22,8 @@ pub enum SignerCommand {
 pub enum SignerEvent {
     /// Signals that a P2P event has occurred.
     P2P(P2PEvent),
+    /// Signals that a block observer event has occurred.
+    BlockObserverUpdated,
```

Comment:
I would rename this to `BitcoinBlockObserved`.

Reply:
Ah perfect, was fumbling after a good name 😊

Reply:
Updated in [e7064d1](https://github.com/stacks-network/sbtc/pull/580/commits/e7064d1c891fc1e5d30906dc33ba69bed4970f6e)